hooks/hook-example.py

```diff
@@ -64,6 +64,23 @@ def get_tasks(secret):
 #   ...
 # ]
 #
+# The custom code in compute_labels should loop and label each one of them and 
+# return a dictionary of the form
+#  
+# {
```

Comment:
@mdipierro so that the dictionary keyed by timestamp isn't the only possible payload, how about we put that map into a "labels" field like:
```json
{
  "labels": {
    "2017-01-01T12:00:00.000": {
      "cat": {"probability: 0.93, "polygon": []},
      "dog": {"probability: 0.88, "polygon": []},
    }
  }
}